src/global_distrib/mod_global_distrib_utils.erl

```diff
@@ -114,6 +114,15 @@ opt(Module, Key) ->
             error(atom_to_list(Module) ++ " required option unset: " ++ atom_to_list(Key))
     end.
 
+-spec opt(module(), Key :: atom(), Default :: term()) -> Value :: term().
+opt(Module, Key, Default) ->
```

Comment:
The reason there was no `opt/3` with default value is that each module specifies its default values on start.

Example: `mod_global_distrib_bounce` adds default values for `resend_after` and `max_retries`: https://github.com/esl/MongooseIM/blob/master/src/global_distrib/mod_global_distrib_bounce.erl#L43

The point was to avoid `try ... catch` because options are consulted very often, and to enable fast inspection and change of all options used by the module (`ets:tab2list(mod_global_distrib_bounce)`).